test_util/aws.py

```diff
@@ -349,10 +333,11 @@ def delete(self):
         self.stack = self.boto_wrapper.resource('cloudformation').Stack(self.stack.stack_id)
         self.stack.delete()
 
+    @retry_boto_rate_limits
     def get_vpc_host_ips(self):
         reservations = self.boto_wrapper.client('ec2').describe_instances(Filters=[{
```

Comment:
I cannot currently think of a more elegant method. At the boto_wrapper level, we just return clients. The issue is a lot (but not all) functions of the clients need to be wrapped for this case.

What I could imagine is changing a client's getattr so that whenever it returns a function it returns a function wrapped with the retry and backoff method, but that feels like a hack